src/Http/Controller/Admin/CategoryController.php

```diff
@@ -274,6 +274,7 @@ public function import(FormBuilder $builder, FileRepositoryInterface $fileReposi
         if (request()->action == "save" and $file = $fileRepository->find(request()->file)) {
             if ($file->extension === 'xls' || $file->extension === 'xlsx') {
                 $pathToFolder = "/storage/streams/" . app(Application::class)->getReference() . "/files-module/local/ads_excel/";
+                app()->register(ExcelServiceProvider::class);
```

Comment:
we can load here @serdarekremcakir

Reply:
please add CatsServiceProvider.php. You can use `protected $provider  = []`;